### PR TITLE
Stage additional ids as sent from the server.

### PR DESCRIPTION
**Pull request by jmchilton**
It is fine if in the future we want to think about disabling the server from sending this - but as long as the GUI elements are connected up I think we should respond to them properly.

### CODE REVIEW

**Comment by jmchilton** [comment]
~~I guess we also need to check if the data is already there - huh? And we should maybe try to replicate the naming pattern used when mounted as a volume? Hmm...~~

Updated my Galaxy PR to now set this environment variable unless the deployer indicates volumes are unavailable.

File: get_notebook.py [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
     shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
+
+additional_ids = os.environ.get("ADDITIONAL_IDS", "")
+if additional_ids:
+    gi = galaxy_ie_helpers.get_galaxy_connection(history_id=history_id, obj=False)
+    hc = HistoryClient(gi)
+    history = hc.show_history(history_id, contents=True)
+    additional_ids = additional_ids.split(",")
+    for additional_id in additional_ids:
+        for hda in history:
```
Comment (bgruening) [inline get_notebook.py]: iterating at first over the history and later over additional_ids should be faster, isn't it?

Comment (jmchilton) [reply get_notebook.py]: Sure - I'll modify this to do that.

**Comment by jmchilton** [comment]
I redid that loop and tested - the other way was clearly suboptimal - thanks for the catch @bgruening!

**Pull request merged**